core/controllers/editor.py

```diff
@@ -101,9 +101,45 @@ def _require_valid_version(version_from_payload, exploration_version):
             % (exploration_version, version_from_payload))
 
 
+class EditorLogoutHandler(base.BaseHandler):
+    """Handles logout from editor page."""
+
+    @acl_decorators.open_access
+    def get(self):
+        """Checks if exploration is published and redirects accordingly."""
+
+        url_to_redirect_to = str(self.request.get('return_url'))
+        if url_to_redirect_to:
+            exploration_id = url_to_redirect_to.split('/')[-1]
```

Comment:
Sorry -- I just realized that this would be cleaner if we matched it against a regex representing the expected return_url structure. Can we do that? We shouldn't be processing stuff like /sadjklashkjsdhfdshkg/possible_exp_id.

Reply:
@seanlip i have changed path to relative style. i'm using 
```
url_to_redirect_to_regex = (
        	r'%s/(?P<explotation_id>[\w-]+)$' % feconf.EDITOR_URL_PREFIX)
```
regex to match to '/create/exp_id'. is there any character missing for exp_id e.g i think "[\w-]+" should match to every exp_id. can any other characters appear.